website_event_snippet_calendar/__manifest__.py

```diff
@@ -15,8 +14,8 @@
         "website_event",
     ],
     "data": [
-        "templates/assets.xml",
-        "templates/embed.xml",
-        "templates/snippets.xml",
+        "views/assets.xml",
+        "views/embed.xml",
+        "views/snippets.xml",
```

Comment:
It's common to split among views (for backend) and templates (qweb, for frontend). I think these were better under templates, don't you think?

FTR see the sample module: https://github.com/OCA/maintainer-tools/tree/master/template/module